### PR TITLE
Improve course API logging.

### PR DESCRIPTION
**Pull request by polesye**
@den99f please review.

### CODE REVIEW

File: common/djangoapps/labster/course/views.py [hunk 1]
```diff
@@ -306,9 +306,12 @@ def course_handler(request, course_key_string=None):
                 delete_course_and_groups(course_key, request.user.id)
                 return Response(status=status.HTTP_204_NO_CONTENT)
 
-    except LicenseStoreError:
-        raise HttpResponseBadRequest({
-            'ErrMsg': 'Incorrect license format.'
+    except LicenseStoreError as err:
+        log.exception(
+            'Incorrect licenses format: %s - %r', request.json.get('licenses', None), err
```
Comment (denys-zarubin) [inline common/djangoapps/labster/course/views.py:311]: I prefer to create before message and than use it in exceptions. 
Like: 

```
msg = 'Incorrect licenses format: %s - %r', request.json.get('licenses', None)
log.exception(msg), err
return HttpResponseBadRequest({"ErrMsg": msg}) 
```

Comment (polesye) [inline common/djangoapps/labster/course/views.py:311]: we have a bit different messages in the log and resonse. So, I do not see any reason to use variable here.

**Comment by denys-zarubin** [comment]
:+1:

File: common/djangoapps/labster/course/views.py (hunk 1 above)
Comment (denys-zarubin) [inline common/djangoapps/labster/course/views.py:311]: Than everything OK.

**Pull request merged**